content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,188 @@
+
+Some other characteristics of ubiquitous language are:
+ 
+* terms have a single meaning within a given context (no exceptions)
+* agreed upon by stakeholders, but not unilaterally defined by them
+* evolves over time
+* consists of domain terms, not technical ones 
+  *  ("linked list", "hashmap", and "singleton" don't generally appear in a ubiquitous language)
+
+There are a lot of "rules", so to speak, to defining and maintaining a ubiquitous language.  You may be asking yourself, "Why go through the effort?" In order to answer that question lets dig into how this could help you and your team deliver better outcomes to the businesses you support.
+
+## The importance
+So why is it important that you all speak the same language?  For decades, huge organizations have adopted the "waterfall" mentality of distilling a business domain into a set of requirements by requirements writers, which were then translated by architects into high level architectures, which were then translated by senior software engineers into lower level software designs, which were then translated and implemented by software engineers on a number of teams throughout the organization.  Ultimately turning the software development process into a huge game of telephone.
+
+!["Hi there you" -> "High fair food" -> "I know you" -> "I know kung fu"](./images/telephone.png)
+
+Some domain terms may have made it from the requirements into the software but a large majority didn't.  This model primarily breaks down when there is an expectation of bidirectional, or skip-level, communication.  The reality is that software development isn't as simple as requirements to architecture to design to code. In order to build software, especially in an "agile" or "lean" manner, especially on a balanced team, you empower all roles to engage with the business. It isn't just the PMs responsibility to learn and understand the business context, but everyone's. In order for this to happen effectively, individuals must communicate.  That communication goes a lot more smoothly if the words that the engineers are using have the same meaning as those translating the business domain to write stories.
+
+Let's use an example to illustrate what you are talking about.  Say that your team is writing a shopping application and your product manager has a user interview with someone from the business and write a single story that says something along the lines of:
```

Comment:
Let's use an example to illustrate what you are talking about.
**should read**
Let's use an example to illustrate what we are talking about.